src/main/webapp/app/entities/car/current.tsx

```diff
@@ -4,13 +4,14 @@ import { Switch } from 'react-router-dom';
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 import ErrorBoundaryRoute from 'app/shared/error/error-boundary-route';
 import { Button, Col, Row } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
-/* jhipster-needle-add-route-import  */
+/* jhipster-needle-add-route-import - JHipster will add routes here */
 import { Modal, ModalFooter, ModalHeader, ModalBody } from 'reactstrap';
-import ReturnCarButton from "app/entities/car/returnButton";
```

Comment:
Dlaczego usunąłeś ten moduł, on może być potrzebny w przyszłości

Reply:
Jest to część mojego refactoringu i funkcjonalność została przeniesiona.